.cicd/docker/amazonlinux-2.dockerfile

```diff
@@ -28,5 +29,5 @@ RUN git clone --single-branch --branch release_80 https://git.llvm.org/git/llvm.
 # ccache
 RUN curl -LO http://download-ib01.fedoraproject.org/pub/epel/7/x86_64/Packages/c/ccache-3.3.4-1.el7.x86_64.rpm && \
     yum install -y ccache-3.3.4-1.el7.x86_64.rpm
-# container entrypoint
-CMD /workdir/.cicd/entrypoint.sh
\ No newline at end of file
+
+CMD echo "Please provide commmands to run..."
```

Comment:
I think these two lines are unnecessary. If we don't have an entrypoint, we don't have an entrypoint.